c2_components/include/mfx_c2_decoder_component.h

```diff
@@ -215,6 +218,11 @@ class MfxC2DecoderComponent : public MfxC2Component
 
     std::vector<std::unique_ptr<C2Param>> m_updatingC2Configures;
 
+    uint64_t m_consumerUsage;
+
+    uint32_t m_surfaceNum;
+    std::list<std::shared_ptr<mfxFrameSurface1>> m_surfacePool; // used in case of system memory
+
     unsigned int m_uOutputDelay = 8u;
-    unsigned int m_uInputDelay = 1u;
+    unsigned int m_uInputDelay = 0u;
```

Comment:
any performance improve if reduce input delay from 1 to 0?

Reply:
Actually, 4 input buffers are enough for decoding https://cactus-source.iind.intel.com/source/xref/CELADON-MASTER-R2/frameworks/av/media/codec2/sfplugin/CCodecBufferChannel.cpp#1011. 
Reduced m_uInputDelay from 2 to 0 is just to save memory for 8K video playback.

Reply:
OK,Thanks